### PR TITLE
add Trippin by Restier and related docs

### PR DESCRIPTION
**Pull request by mirsking**

### CODE REVIEW

File: pages/service-usages/basic-usages.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
+## Basic Usages
+
+[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/522bf43d2b5f9f09f7ee)
+
```
Comment (chinadragon0515) [inline pages/service-usages/basic-usages.md]: I tried this link, the URL in sample are localhost, update to odata.org

File: pages/service-usages/basic-usages.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+For usage of advanced scenarios, please refer to [Trippin Advanced Usages](service-usages/trippin-advanced-usages/).
+
+### Requesting Data
+
+### Requesting Entity Collections
+
```
Comment (chinadragon0515) [inline pages/service-usages/basic-usages.md]: Change Entity Collections to EntitySet to match terminology in standard

File: pages/service-usages/basic-usages.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+```
+
+#### Requesting an Individual Entity by ID
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People('russellwhyte')
+```
+
+#### Requesting Data/Requesting an Individual Property 1/2
+
```
Comment (chinadragon0515) [inline pages/service-usages/basic-usages.md]: Request single property value

File: pages/service-usages/basic-usages.md [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/Airports('KSFO')/Name 
+```
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/Airports('KSFO')/Location/Address 
+```
```
Comment (chinadragon0515) [inline pages/service-usages/basic-usages.md]: Add one more without Address which just return a complex property

Also add request to return a collection

File: pages/service-usages/basic-usages.md [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+
+#### Requesting an Individual Property Raw Value
+
```
Comment (chinadragon0515) [inline pages/service-usages/basic-usages.md]: Requesting an single primitive or enum type Property Raw Value, we do not support to request structural type property value or collection.

File: pages/service-usages/basic-usages.md [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+### Querying Data
+
+#### System Query Option $filter
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People?$filter=FirstName eq 'Scott'
+```
+
+#### Filter on Complex Type
+
```
Comment (chinadragon0515) [inline pages/service-usages/basic-usages.md]: Complex Type property

File: pages/service-usages/basic-usages.md [hunk 7]
```diff
@@ -0,0 +1,209 @@
...
+```
+
+#### System Query Option $count
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People/$count
+```
+
+#### System Query Option $expand
+
```
Comment (chinadragon0515) [inline pages/service-usages/basic-usages.md]: Add some case with $expand has query options too, like $expand=Friends($select= ..., $top=)

File: pages/service-usages/basic-usages.md [hunk 8]
```diff
@@ -0,0 +1,209 @@
...
+```
+
+#### System Query Option $select
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/Airports?$select=Name, IcaoCode
+```
+
+#### Lambda Operators
+
```
Comment (chinadragon0515) [inline pages/service-usages/basic-usages.md]: Move this up with $filter, and change name to filter on collection

File: pages/service-usages/basic-usages.md [hunk 9]
```diff
@@ -0,0 +1,209 @@
...
+```
+PATCH http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People('russellwhyte')
+header:
+{
+	Content-Type: application/json
+}
+body:
+{
+    "@odata.type":"Microsoft.OData.Service.Sample.TrippinInMemory.Models.Person",
+    "FirstName": "Mirs",
```
Comment (chinadragon0515) [inline pages/service-usages/basic-usages.md]: @odata.type is not needed here, remove it.

File: pages/service-usages/basic-usages.md [hunk 10]
```diff
@@ -0,0 +1,209 @@
...
+}
+body:
+{
+    "userName": "scottketchum",
+    "tripId": 0
+}
+```
+
+### ETag
+
```
Comment (chinadragon0515) [inline pages/service-usages/basic-usages.md]: Etag support

File: pages/service-usages/basic-usages.md [hunk 11]
```diff
@@ -0,0 +1,209 @@
...
+##### Update Entity with ETag
```
Comment (chinadragon0515) [inline pages/service-usages/basic-usages.md]: Add first request, get etag, this is common user scenario, get data first and get etag.
And for etag, indicates two part, one in body and one in header.

File: pages/service-usages/advanced-usages.md [hunk 12]
```diff
@@ -0,0 +1,180 @@
...
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People/Microsoft.OData.Service.Sample.TrippinInMemory.Models.Employee
+```
+
+#### Filter on Derived Entity
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People('kristakemp')/Microsoft.OData.Service.Sample.TrippinInMemory.Models.Employee?$filter=UserName eq 'mirsking'
+```
```
Comment (chinadragon0515) [inline pages/service-usages/advanced-usages.md]: Add filter on property belongs to derived type only

File: pages/service-usages/advanced-usages.md [hunk 13]
```diff
@@ -0,0 +1,180 @@
...
+    "LastName": "King",
+    "AddressInfo": [
+    {
+      "Address": "Microsoft",
+      "City": {
+        "Name": "Shanghai",
+        "CountryRegion": "China"
+      }
+    }
+    ]
```
Comment (chinadragon0515) [inline pages/service-usages/advanced-usages.md]: Do we have derived type properties here? Have some property easier to know belongs to Employee like EmployeeSeat or something else.

File: pages/service-usages/advanced-usages.md [hunk 14]
```diff
@@ -0,0 +1,187 @@
...
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/Me/AddressInfo
+```
+
+### Derived Entity Type
+
+#### Requesting Derived Entity Type
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People('kristakemp')/Microsoft.OData.Service.Sample.TrippinInMemory.Models.Employee
```
Comment (VikingsFan) [inline pages/service-usages/advanced-usages.md]: can we remove the key 3mslpb2bc0k5ufk24olpghzx here?

File: index.html [hunk 15]
```diff
@@ -12,14 +12,14 @@
     <!-- Wrapper for slides -->
     <div class="carousel-inner zero-side-margin" role="listbox">
       <div class="item active">
-        <img src="/assets/homepage_1.jpg" alt="Home">
```
Comment (VikingsFan) [inline index.html]: why we edit this file

File: pages/reference-service.html [hunk 16]
```diff
@@ -18,7 +21,23 @@
...
-  <div class="tab-pane active" id="v4">
+  <div class="tab-pane active" id="v4-restier">
+    <table class="table table-striped table-condensed table-hover">
+      <tr>
+        <th>Name</th>
+        <th>Description</th>
+      </tr>
+      <tr>
+        <td><a href="http://services.odata.org/TripPinRESTierService">TripPin(read/write)</a></td>
+        <td>The new OData V4 service with <a href="http://odata.github.io/RESTier/">Restier</a> designed for real scenarios and covering most V4 features.</td>
```
Comment (VikingsFan) [inline pages/reference-service.html]: OData V4

**Pull request merged**